modules/classic_ui/pages/session_management.adoc

```diff
@@ -1,11 +1,10 @@
 = Session Management
 :toc: right
+:description: The personal settings page allows you to have an overview of the connected browsers and clients. It is accessed by selecting the
```

Comment:
By selecting the what???

Reply:
I fix it before merging, thanks for catching 👍